extensions/functions_arithmetic.yaml

```diff
@@ -1369,6 +1369,109 @@ aggregate_functions:
         ordered: true
         return: LIST?<any>
 
+  - name: "bit_or"
+    description: >
+      Calculates the bitwise OR of all values in set of values.
+    impls:
+      - args:
+          - name: x
+            value: i8
+        nullability: DECLARED_OUTPUT
```

Comment:
`DECLARED_OUTPUT` means that the input *can* be nullable. So how does this function deal with nulls? I imagine it ignores them, but this should be documented.